modin/experimental/pandas/__init__.py

```diff
@@ -48,6 +50,19 @@
     to_pickle_distributed,
 )
 
+old_to_pickle_distributed = to_pickle_distributed
+
+
+@functools.wraps(to_pickle_distributed)
+def to_pickle_distributed(*args, **kwargs):
+    warnings.warn(
+        "`DataFrame.to_pickle_distributed` will be removed after Modin 0.26.0 release. "
```

Comment:
```suggestion
        "`DataFrame.to_pickle_distributed` is deprecated and will be removed in a future version. "
```

Reply:
Why don't you want to specify an explicit release?

Reply:
Our plans may change. Pandas also does not specify an exact release to deprecate functionality.

Reply:
changed